### DISCUSSION TITLE
[Help] Escaping

### DISCUSSION THREAD

**Discussion by kiil** [zk-org/zk#334] [Help] [answered]
### Verify

- [X] I searched the existing discussions for help

### How can we help you?

I use Hugo and Hugo uses shortcodes in the form of 

```
{{% mdshortcode %}}Stuff to `process` in the *center*.{{% /mdshortcode %}}
```
and

```
{{< highlight go >}} A bunch of code here {{< /highlight >}}
```

Either way, this seems to cause conflict with handlebars in zk.

According to handlebar docs, escaping should be possible to do like so for inline

```
\{{escaped}} 
```
and like so for blocks

```
{{{{raw}}}}
  {{escaped}}
{{{{/raw}}}}
```

But it seems zk will still not just ignore the shortcodes from hugo.

I came up with a working solution like so

```
{{ concat '{{< lead >}}' '' }}{{ concat '{{< param description>}}' '' }}.{{ concat '{{< /lead >}}' '' }}
```

But that seems to be a little complicated.

Is there an easier way?

**Comment by mickael-menu** [comment, marked as answer]
I tried with this template:

(note the `\` in front of both `{{`)

```toml
# {{title}}

\{{% mdshortcode %}}Stuff to `process` in the *center*.\{{% /mdshortcode %}}
```

And it properly expanded as:

```toml
# Untitled

{{% mdshortcode %}}Stuff to `process` in the *center*.{{% /mdshortcode %}}
```

**Reply by kiil** [reply]
Thank you. It works. I must have left out the second backslash before.
